shared/src/common/enums/enums.ts

```diff
@@ -2,5 +2,5 @@ export { ApiPath, AuthApiPath } from './api/api';
 export { CustomExceptionName } from './exceptions/exceptions';
 export { ContentType } from './file/file';
 export { HttpCode, HttpHeader, HttpMethod } from './http/http';
-export { UserValidationMessage } from './user/user';
+export { UserValidationMessage, UserSignInValidationRule } from './user/user';
```

Comment:
```suggestion
export { UserValidationMessage } from './user/user';
```
all rules should be into one enum